static/app/components/tag.tsx

```diff
@@ -97,56 +120,20 @@ function BaseTag({
     </Tooltip>
   );
 
-  function handleDismiss(event: React.MouseEvent) {
-    event.preventDefault();
-    onDismiss?.();
-  }
-
-  const trackClickEvent = () => {
-    trackAnalytics('tag.clicked', {
-      is_clickable: defined(onClick) || defined(to) || defined(href),
-      organization: null,
-    });
-  };
-
-  function tagIcon() {
-    if (isValidElement(icon)) {
-      return <IconWrapper>{cloneElement(icon, {...iconsProps})}</IconWrapper>;
```

Comment:
We can do this with the `IconDefaultsProvider` now without the use of clone element.